src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -10,15 +10,29 @@ public class WorkWithFile {
     public static final String NAME_OF_SUPPLY = "supply";
     public static final int OPERATION_INDEX = 0;
     public static final int AMOUNT_INDEX = 1;
+    public static final String REGEX_SPLIT = "[,\\s]";
+    public static final String STATISTIC_PATTERN = "supply,%d%n"
+            + "buy,%d%n"
+            + "result,%d%n";
+    public static final int INITIAL_SUPPLY_AMOUNT = 0;
+    public static final int INITIAL_BUY_AMOUNT = 0;
+    private String toWrite;
+    private int supplyAmount;
+    private int buyAmount;
 
     public void getStatistic(String fromFileName, String toFileName) {
-        String toWrite;
+        read(fromFileName);
+        calculateStatistic();
+        write(toFileName);
+    }
+
+    private void read(String fromFileName) {
```

Comment:
read should read data from file and return String as a result. [Use String builder to concat all file lines](https://mate-academy.github.io/jv-program-common-mistakes/java-core/builder-file/work-with-file#use-stringbuilder-instead-of-string-to-concat-data-in-loops).